components/NavBar.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from './header/Header.module.scss'
+import Link from './Link'
+import useUser from '../libs/use-user'
+import { useRouter } from 'next/router'
+import axios from 'axios'
+
+const NavBar = () => {
+
+  const { user, mutateUser } = useUser() // check user is authenticated
+  const router = useRouter()
+
+  return (
+    <nav className={styles.navbar}>
+      <div className={styles.navbarContainer}>
```

Comment:
```suggestion
      <div className={styles['nav-container']}>
```

same for other place